pkg/aspect/config/quotas.proto

```diff
@@ -32,7 +33,15 @@ message QuotasParams {
         // Map of quota_descriptor label name to attribute expression. At run time each
         // expression will be evaluated to determine the value provided to the aspect. The
         // result of evaluating the expression must match the ValueType of the label in the quota_descriptor.
-        map<string, string> labels = 3;
+        map<string, string> labels = 2;
+
+        // The upper limit for this quota.
+        int64 max_amount = 3;
```

Comment:
I think it should be named `amount` (i.e. ratelimit).    (`max_amount` means the same thing in this context)
We have an orthogonal mechanism to decide which limit finally applies.

This message is what one authority (modeled as scope) would like the quota to be.